app/models/user.rb

```diff
@@ -9,7 +9,8 @@ class User < ActiveRecord::Base
   validates :github_username, presence: true, uniqueness: true
   validates :uid, presence: true
 
-  attr_encrypted :token, key: 'hello'
+  attr_encrypted :token, key: '8rnah8l2910knalah8gaHOGSs#%3WWgh'
```

Comment:
is there any concern about this secret being in github?

Reply:
I'm honestly not sure and was wondering the same thing.  I know a key base is set via environment variable here https://github.com/appfolio/ladle/blob/master/config/secrets.yml, but I should look into whether that is only used for devise.

Reply:
A search for `secret_key_base` only returned an issue where they decided not to use it as the default for the key.

Reply:
my gut feeling is it might be a problem to check this in (even though the bogus key was already checked in before) I know we have had to do a lot of clean up lately around checked in secrets and we are trying to not check anything in anymore. I think it would be good to throw a quick question into #security to see how they feel about those af_encrypted_attribibute keys. I did a quick search in apm but could not find an example there, seems like we stopped using it there.

Reply:
ideally not, this is a public repo right?

Reply:
Oh yes, it's public. 🙂